west.yml

```diff
@@ -49,6 +49,7 @@ manifest:
       path: modules/hal/silabs
     - name: hal_st
       revision: b6c9262eed68000e69c92bef6e820cdbd5b0a32b
```

Comment:
need to remove the old revision, we need only one.

Reply:
Of course... Sorry, forgot it.